src/platform/plugins/shared/ai_assistant_management/selection/public/components/navigation_control/index.tsx

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React, { useState, useCallback } from 'react';
+import {
+  EuiModal,
+  EuiOverlayMask,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiButton,
+  EuiCard,
+  EuiIcon,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  useGeneratedHtmlId,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiLink,
+  EuiText,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import type { CoreStart } from '@kbn/core/public';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { PREFERRED_AI_ASSISTANT_TYPE_SETTING_KEY } from '../../../common/ui_setting_keys';
+import { AIAssistantType } from '../../../common/ai_assistant_type';
+import { AssistantIcon } from '../../icons/assistant_icon/assistant_icon';
+
+interface AiAssistantHeaderButtonProps {
+  coreStart: CoreStart;
+  isSecurityAIAssistantEnabled: boolean;
+  isObservabilityAIAssistantEnabled: boolean;
+  triggerOpenChat: (event: { assistant: AIAssistantType }) => void;
+}
+
+export const AiAssistantHeaderButton: React.FC<AiAssistantHeaderButtonProps> = ({
```

Comment:
super duper nit: Suggestion to make this consistent with the rest with AIAssistant... (so upper cased I and instead of lower-cased)